src/Core/helpers.php

```diff
@@ -565,3 +565,30 @@ function web_path($path = '')
         return app()->webPath($path);
     }
 }
+
+if (! function_exists('asset')) {
+    /**
+     * Generate an asset path for the application.
+     *
+     * @param  string  $path
+     * @param  bool    $secure
+     * @return string
+     */
+    function asset($path, $secure = null)
+    {
+        return app(UrlGenerator::class)->asset($path, $secure);
```

Comment:
@Jaspervv I'm ok for implementation but can you just update the `asset` function to use the alias name instead of the class name `UrlGenerator` ?

Also, helpers functions are organized in alphabetical order by name, can you also move the `secure_asset` function below ?

Thanks

Reply:
Sure, no probs. Is it okay like this?  
I did it with the namespace because that's how the `url()` helper does it.

Reply:
I'll probably have to clean my own helpers as well 😄 Thanks for the change.